roles/reboot/tasks/main.yml

```diff
@@ -17,6 +17,21 @@
 - set_fact:
     real_ansible_host: "{{ ansible_host }}"
     timeout: "{{ cli_reboot_timeout | default('120') }}"
+    wf_delay: "{{ cli_wf_delay | default('30') }}"
```

Comment:
I just tested this with the no cli_wf_delay passed in and it worked nicely!  Is there still a need to be able to pass in a wf_delay with the regex_search?

Reply:
Aside: I actually tested with `cli_wf_delay=10` and it still worked well, so the regex is proving useful.

I'm not opposed to dropping out the option to configure the delay.